docs/changelog.rst

```diff
@@ -1,8 +1,24 @@
 :orphan:
 
+.. See https://releases.readthedocs.io/ for instructions
+
 =========
 Changelog
 =========
+
+* :release:`3.2.0 <2020-06-??>`
```

Comment:
Leaving the date unset it's released.

Reply:
@sigmavirus24 I just saw your reaction; GitHub doesn't send notifications for those.

Can you elaborate, or offer a suggestion? I realize it's sub-optimal, but I wanted to have the release in the changelog as part of the pre-release, without committing to a specific final release. I figure this is a one-time thing while I walk through my first release.

For now, I've changed it to `<unreleased>`.